api/src/models/priceHistory.model.ts

```diff
@@ -0,0 +1,27 @@
+import { model, Schema, Document } from 'mongoose';
+import { PriceHistoryData } from '@interfaces/priceHistory.interface'; 
+
+const PriceHistorySchema: Schema = new Schema({
+  month: {
```

Comment:
Approving this PR, however, typically what you will see is that M/D/Y is usually a format that can be done with either a date or time object. https://www.mongodb.com/docs/manual/reference/method/Date/